Makefile

```diff
@@ -1,23 +1,33 @@
 COMPOSE = docker-compose
-DEVDOCKER = $(COMPOSE) exec web
 
 build:
-	$(COMPOSE) build web
+	$(COMPOSE) build
 
 web:
-	rm -fr static/*  # Workaround for whitenoise busyness in dev
 	$(COMPOSE) up web
 
+enter:
+	$(COMPOSE) exec web bash
+
 compilescss:
-	$(DEVDOCKER) web ./manage.py compilescss
-	rm -fr static/*
-	$(DEVDOCKER) web ./manage.py collectstatic --noinput
+	$(COMPOSE) exec web python manage.py compilescss
+	$(COMPOSE) exec web python manage.py collectstatic --clear --noinput
 
 update_index:
-	$(DEVDOCKER) web ./manage.py update_index
+	$(COMPOSE) exec web python manage.py update_index
```

Comment:
Default wagtail `update_index` only indexes pages. We needed ability to index page components separately down to a widget. Hence we created our own indexing command `update_topics_index`.